prombench/manifests/prombench/benchmark/6_loadgen.yaml

```diff
@@ -48,6 +48,28 @@ data:
         - expr: histogram_quantile(0.99, sum by(path, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(path, method, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
         - expr: histogram_quantile(0.99, sum by(instance, le) (rate(codelab_api_request_duration_seconds_bucket{method="POST"}[5m])))
+      - name: arthematic_operation
+        interval: 10s
+        type: instant
+        queries:
+        - expr: sum(node_cpu_seconds_total)/sum(container_memory_rss)
+        - expr: rate(node_cpu_seconds_total[5m]) * 5
+        - expr: sum(go_gc_heap_goal_bytes)/sum(loadgen_query_duration_seconds_created)
```

Comment:
This has the same structural problem as the first one.

Reply:
I replaced the query with `sum by(instance) (rate(node_cpu_seconds_total[5m])) / avg by(instance) (node_load1)`, which is also structured with the label instance.